optuna/cli.py

```diff
@@ -70,16 +68,38 @@ def get_parser(self, prog_name: str) -> ArgumentParser:
             help="If specified, the creation of the study is skipped "
             "without any error when the study name is duplicated.",
         )
+        parser.add_argument(
+            "--directions",
+            type=str,
+            default=None,
+            help="Set directions of optimization to a new study."
+            " Put comma between directions. Each direction should be"
+            " either `minimize` or `maximize`.",
+        )
         return parser
 
     def take_action(self, parsed_args: Namespace) -> None:
 
+        if parsed_args.direction is not None:
+            if parsed_args.directions is not None:
+                raise ValueError("Specify only one of `direction` and `directions`.")
+            else:
+                directions = [parsed_args.direction]
+        else:
+            if parsed_args.directions is not None:
+                directions = parsed_args.directions.split(",")
+            else:
+                directions = ["minimize"]  # default value
+
+        if any(direction not in ("maximize", "minimize") for direction in directions):
+            raise ValueError("Each direction must be `minimize` or `maximize`")
```

Comment:
Is this error handling not taken care of already inside `create_study`?

Reply:
Exactly, thank you. I dropped the validation in 888d64e.